src/eterna/pose2D/Base.ts

```diff
@@ -588,13 +587,12 @@ export default class Base extends ContainerObject implements LateUpdatable {
     }
 
     private static multiplyAlphaFilter(multiplier: number): ColorMatrixFilter {
-        const filter = new filters.ColorMatrixFilter();
+        const filter = new ColorMatrixFilter();
         filter.matrix = [
             1, 0, 0, 0, 0,
             0, 1, 0, 0, 0,
             0, 0, 1, 0, 0,
-            0, 0, 0, multiplier, 0,
-            0, 0, 0, 0, 1
+            0, 0, 0, multiplier, 0
```

Comment:
why is this matrix now 4x5 instead of 5x5?

Reply:
Plot twist, it was always 4x5! It just wasn't typechecked till now, and the last "row" was ignored by Pixi

Reply:
lmao